frontend/packages/operator-lifecycle-manager/src/components/catalog-source.tsx

```diff
@@ -431,18 +431,18 @@ const DisabledPopover: React.FC<DisabledPopoverProps> = ({ operatorHub, sourceNa
   const { t } = useTranslation();
   return (
     <PopoverStatus
-      title={t('catalog-source~Disabled')}
+      title={t('olm~Disabled')}
       isVisible={visible}
       shouldClose={close}
-      statusBody={<StatusIconAndText title={t('catalog-source~Disabled')} />}
+      statusBody={<StatusIconAndText title={t('olm~Disabled')} />}
     >
       <p>
         {t(
-          'catalog-source~Operators provided by this source will not appear in OperatorHub and any operators installed from this source will not receive updates until this source is re-enabled.',
+          'olm~Operators provided by this source will not appear in OperatorHub and any operators installed from this source will not receive updates until this source is re-enabled.',
```

Comment:
disable catalog source modal message and button are not completely translated
<img width="450" alt="Screen Shot 2020-12-15 at 6 58 08 PM" src="https://user-images.githubusercontent.com/12692381/102206433-9316d480-3f07-11eb-8f06-8b32129cb7e4.png">

Reply:
See #7499